aiosmtpd/tests/test_handlers.py

```diff
@@ -409,11 +409,11 @@ def test_deliver(self):
             mock().connect.assert_called_once_with('localhost', 9025)
             mock().sendmail.assert_called_once_with(
                 'anne@example.com', ['bart@example.com'], """\
-From: Anne Person <anne@example.com>
-To: Bart Person <bart@example.com>
-Subject: A test
-X-Peer: ::1
-
+From: Anne Person <anne@example.com>\r
+To: Bart Person <bart@example.com>\r
+Subject: A test\r
+X-Peer: ::1\r
+\r
```

Comment:
Thats tricky case ) Whole file have Linux style LF endings, so when message is about to send it have linux endings. But when SMTP library send it, it do _fix_eols and thats changes line endings to CRLF. So "preserving" expected behavior is have CRLF here. But as file is LF I added CR at line ends.